### DISCUSSION TITLE
All buildings and structures in the default scenery have disappeared

### DISCUSSION THREAD

**Discussion by Alexfx82** [kubilus1/autoortho#204] [Q&A]
good day! I installed Autoortho according to the instructions, https://kubilus1 .github.io/autoortho/!
But in default airports, all the buildings disappeared from me, only the equipment and taxiways and rwy remained, and how can this be fixed! tnx

**Comment by NNowakowski** [comment]
Check your scenery_packs.ini file.
You want to have all the autoortho directories at the bottom like this.
```
...
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Washington DC/
SCENERY_PACK *GLOBAL_AIRPORTS*
SCENERY_PACK Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_autoortho/
SCENERY_PACK Custom Scenery/z_afr_00/
SCENERY_PACK Custom Scenery/z_afr_01/
...
```

**Reply by Alexfx82** [reply]
Thanks for the answer! I did everything as you wrote, but nothing has changed, default airports without buildings

**Comment by Alexfx82** [comment]
I found the reason! It's all about ..\simHeaven_X-World.. It is these libraries that remove the surrenders

**Reply by TinkerNZ** [reply]
Provided you are using the latest version of simheaven_X-World, this should not remove anything (exclusions folder has been removed).
And if you have the ini order as recommended by the Dev. everything should display correctly. (see my edited order below).
Also the Dev. recommended to have the z_autoortho folder **below** the base pacs. Further he mentioned there is no need to use the yAutoOrtho_Overlays if you are using the simheaven Network 8 overlay.

...
SCENERY_PACK *GLOBAL_AIRPORTS*
SCENERY_PACK Custom Scenery/X-Plane Landmarks - Sydney/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-1-vfr/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-2-regions/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-3-details/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-4-extras/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-5-footprints/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-6-scenery/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-7-forests/
SCENERY_PACK Custom Scenery/simHeaven_X-World_Australia-Oceania-8-network/
SCENERY_PACK_DISABLED Custom Scenery/yAutoOrtho_Overlays/
SCENERY_PACK Custom Scenery/z_aus_pac_00/
SCENERY_PACK Custom Scenery/z_aus_pac_01/
SCENERY_PACK Custom Scenery/z_aus_pac_02/
SCENERY_PACK Custom Scenery/z_aus_pac_03/
SCENERY_PACK Custom Scenery/z_aus_pac_04/
SCENERY_PACK Custom Scenery/z_aus_pac_05/
SCENERY_PACK Custom Scenery/z_aus_pac_06/
SCENERY_PACK Custom Scenery/z_aus_pac_07/
SCENERY_PACK Custom Scenery/z_aus_pac_08/
SCENERY_PACK Custom Scenery/z_aus_pac_09/
SCENERY_PACK Custom Scenery/z_aus_pac_10/
SCENERY_PACK Custom Scenery/z_aus_pac_11/
SCENERY_PACK Custom Scenery/z_aus_pac_12/
SCENERY_PACK Custom Scenery/z_aus_pac_13/
SCENERY_PACK Custom Scenery/z_autoortho/

**Reply by Alexfx82** [reply]
Thank you very much!!
Did I understand correctly that I can remove yAutoOrtho_Overlay?

**Reply by TinkerNZ** [reply]
Strictly speaking you can if you are using SimHeaven network 8 overlay, however I suggest you disable it either manually as shown above or as in my case I use xOrganizer and I just untick/disable it with that.
Reason why removing the yAutoOrtho_Overlays folder is not the best option is that when you ad another region or update an installed pack with AO, it will be recreated again. If you 'DISABLE' it any add-ons or updates will just go into that folder but won't be used.
